cmd/snap/cmd_prepare_image.go

```diff
@@ -110,7 +111,7 @@ For preparing classic images it supports a --classic mode`),
 }
 
 var imagePrepare = image.Prepare
-var imageReadSeedManifest = image.ReadSeedManifest
+var seedReadSeedManifest = seedwriter.ReadSeedManifest
```

Comment:
suggestions for naming here taken, it sounds weird to me to repeat 'seed'

Reply:
Nice suggestion and makes a ton of sense - I'll do that

Reply:
I did this in a follow up to keep this as atomic changes: https://github.com/snapcore/snapd/pull/12732